deploy/environments/libraries.yaml

```diff
@@ -27,6 +27,7 @@ spack:
     - boost+python@1.70.0
     - boost+python@1.73.0
     - catch2
+    - cli-tools
```

Comment:
If you add it to `external` modules, this is the wrong place :)

Should also really be in `externals`, shouldn't it?

Reply:
indeed, sorry for overlooking it